shims/spark311cdh/src/main/scala/com/nvidia/spark/rapids/shims/spark311cdh/SparkShimServiceProvider.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.shims.spark311cdh
+
+import com.nvidia.spark.rapids.{ClouderaShimVersion, SparkShims}
+
+object SparkShimServiceProvider {
+  val VERSION = ClouderaShimVersion(3, 1, 1, "3.1.7270")
+  val VERSIONNAMES = Seq(s"3.1.1.3.1.7270")
```

Comment:
so what happens when you put out an update for Spark (thinking bug fix), you will also put out an updated plugin version?